src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { ArrowHelper, Color, ColorRepresentation, Vector3 } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  //self
+  export let dir:Vector3;
+  export let origin:Vector3;
+  export let length:number = 1;
+  export let color:ColorRepresentation = "#ffff00";
+  export let headLength:number = 0.2 * length;
+  export let headWidth:number = 0.2 * headLength;
+
+  let helper: ArrowHelper = new ArrowHelper(dir,origin,length,color,headLength,headWidth);
+
+</script>
+
+{#if helper}
+  <Object3DInstance object={helper} />
```

Comment:
The [source](https://github.com/mrdoob/three.js/blob/master/src/helpers/ArrowHelper.js) states that the ArrowHelper extends THREE.Object3D and as such the usage of the component `<Object3DInstance>` is a good choice.
Pass down all props that also an Object3D can have. See the types of other components for reference. Outsource the types to the components.ts and extend on the types `Object3DInstanceProperties`. It's not a perfect system but it's somewhat coherent.
I still hope for the svelte language-tools to improve the situation as passing down props with {...$$props} with type support would make everything so much easier.